### PR TITLE
Added hand_delivery flag

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Added the hand_delivery flag in casev2.cases

# What has changed

- Added the hand_delivery flag in casev2.cases. 

# Checklist
* [x] Updated patch number
* [x] Updated version_tag

# How to test?

- Apply groundzero and it should come up with hand_delivery flag
# Links

https://trello.com/c/774V6fGU/580-add-handdelivery-flag-to-casecreated-and-caseupdated-events-and-case-api-8

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[add handDelivery flag to caseCreated and caseUpdated events and case API (8)](https://trello.com/c/774V6fGU/580-add-handdelivery-flag-to-casecreated-and-caseupdated-events-and-case-api-8)

File: patches/600_add_new_hand_delivery_column.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- ****************************************************************************
+-- *** MANUAL RM SQL DATABASE UPDATE SCRIPT                                 ***
+-- ****************************************************************************
+-- *** Number: 600                                                          ***
+-- *** Purpose: Add new "hand_delivery" table to the casev2 schema   ***
+-- *** Author: Ryan Grundy                                                  ***
+-- ****************************************************************************
```
Comment (AdamHawtin) [inline patches/600_add_new_hand_delivery_column.sql]: Not sure these comments are necessary any more? Also they're not strictly manual scripts now we've automated the patching.

File: patches/600_add_new_hand_delivery_column.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+ALTER TABLE casev2.cases
+    ADD COLUMN IF NOT EXISTS hand_delivery boolean NOT NULL DEFAULT FALSE;
```
Comment (AdamHawtin) [inline patches/600_add_new_hand_delivery_column.sql]: Just something to note, when we deploy this to a test environment the default values will be incorrect for some existing cases, need to let the testers know it'll only be correct for samples loaded after the deployment.

File: patches/600_add_new_hand_delivery_column.sql (hunk 1 above)
Comment (NickGrantONS) [reply patches/600_add_new_hand_delivery_column.sql]: agreed

**Pull request merged**